src/main/java/io/milvus/param/IndexType.java

```diff
@@ -27,6 +27,7 @@
  */
 public enum IndexType {
     INVALID,
+    None(0),
```

Comment:
Both INVALID and None are 0. No need to keep INVALID.

Reply:
OK